active_admin_datetimepicker.gemspec

```diff
@@ -19,6 +19,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activeadmin", "~> 1.1"
+  spec.add_dependency "activeadmin", ">= 1.1", "< 3.x"
```

Comment:
If the intention is to not allow future ActiveAdmin 3.0.0 prereleases, then the correct way to do that is:

`spec.add_dependency "activeadmin", ">= 1.1", "< 3.a"`

Reply:
Thank you, my intention was exactly like that